### ISSUE TITLE
Method for getting target and prediction data on video.

### ISSUE THREAD

**Issue by tessavdheiden** [vvanirudh/srnn-pytorch#6]
Hi All,

For everybody who is struggling with getting the target and prediction nodes plotted on the frame and image, I have written a method:
change Dataloader method frame_preprocess(): move skip = 10 up and resample frameList: frameList_data.append(frameList[::skip])
next this method visualized the data in sample.py:
def standard_visualizer(target, prediction, directory, frames, save_directory, batch, epoch):
    file_path_interpolated = os.path.join(directory, 'pixel_pos_interpolate.csv')
    data = np.genfromtxt(file_path_interpolated, delimiter=',')

    if directory == '../data/ucy/zara/zara01' or directory == '../data/ucy/zara/zara02':
        frame_rate = 8
    elif directory == '../data/ucy/univ':
        frame_rate = 4
    else:
        frame_rate = 1

    vidcap = imageio.get_reader(directory + "/seq.avi", 'ffmpeg')
    fps = vidcap.get_meta_data()['fps']
    writer = imageio.get_writer(save_directory  +'/video/epoch_' + str(epoch) +'_batch_'+ str(batch) + '.mp4', fps=fps)

    for t in range(len(prediction)):
        frame_number = int(frames[t + len(prediction) - 4])
        image = vidcap.get_data(int(frame_number * frame_rate))
        plt.imshow(image)

        # plot target
        x_target, y_target = mapper_to_img_coordinates(image.shape[0:2], target[t, :, 1], target[t, :, 0], True)
        colors = plt.cm.gist_ncar(np.linspace(.1, .9, len(x_target)))
        plt.scatter(y_target, x_target, edgecolors='black', marker='o', color=colors)
        
        # plot prediction
        x_pred, y_pred = mapper_to_img_coordinates(image.shape[0:2], prediction[t, :, 1], prediction[t, :, 0], True)
        plt.scatter(y_pred, x_pred, marker='x', edgecolors='black', color=colors)
        plt.axis([0, image.shape[1], image.shape[0], 0])

        plt.draw()
        plt.pause(0.0001)
        plt.savefig(save_directory  +'/video/idx' + ('_%03d.png' % (t)))
        image = plt.imread(save_directory  +'/video/idx' + ('_%03d.png' % (t)))
        writer.append_data(image)
        plt.gcf().clear()

    writer.close()
    plt.close()

last this is required:
def mapper_to_img_coordinates(img_size, x_pos, y_pos, data_negative=False):
    if data_negative:
        new_x = (x_pos + 1) / 2 * img_size[0]
        new_y = (y_pos + 1) / 2 * img_size[1]
    else:
        new_x = x_pos *img_size[0]
        new_y = y_pos * img_size[1]
    return new_x, new_y
